wee_slack.py

```diff
@@ -3693,6 +3727,33 @@ def command_status(data, current_buffer, args):
         s = SlackRequest(team.token, "users.profile.set", {"profile": profile}, team_hash=team.team_hash)
         EVENTROUTER.receive(s)
 
+@utf8_decode
+def line_event_cb(data, signal, hashtable):
+    buffer_pointer = hashtable["_buffer"]
+    line_timestamp = hashtable["_chat_line_date"]
+    line_time_id = hashtable["_chat_line_date_printed"]
+
+    e = EVENTROUTER
+    if line_timestamp and line_time_id and buffer_pointer in e.weechat_controller.buffers:
```

Comment:
I saw now that `line_timestamp` and `line_time_id` is `-1` if you select e.g. a date change line, and that is truthy. When can these values be falsy?

Reply:
They represent date.
time_id is actually a "misuse" of date_printed (cf doc here for recall: https://github.com/wee-slack/wee-slack/blob/master/wee_slack.py#L3147 ) , which is a time (according to the documentation of weechat, if you search for `date_printed` in https://weechat.org/files/doc/stable/weechat_plugin_api.en.html#hdata_line_data)

So this value for non-wee-slack related times should instead be a timestamp that is about equal to nowadays time.

So we could check that it is both positive and less than say 1 million (ierk)

However, in this case, we have a subsequent check that will check that the SlackTS is indeed in the existing message lists, so we need only to make sure that `int(the_value)` (called in the SlackTS initiator) is not failing, which should only be a problem for None.

Reply:
Yeah, I know they represent dates. I just noticed that it's `-1` when not set, so I wondered if the if worked as you indented. But if it's just to handle `None` to prevent errors in `SlackTS`, then it makes sense. I don't really think it will ever be None though, but I'm not sure, so let's keep it.